src/xdocs/google_style.xml

```diff
@@ -66,6 +66,13 @@
         </p>
       </subsection>
       <subsection name="Coverage table" id="Google_Coverage_table">
+        <p>
+          <b>ATTENTION:</b>
+          Links to config and test in following table reference to latest (not released yet) config,
+          it might be slightly different from what we have in latest release.
```

Comment:
To me these should be 2 separate sentences instead of a comma. It may help to explain what `it` is in the second sentence.

Reply:
fixed.
If any other changes should be done, it would be awesome if you provide exact wording.